grid-proxy/docs/database.md

```diff
@@ -0,0 +1,11 @@
+# Database
+
+The grid proxy has access to a postgres database containing information about the tfgrid.\
+The grid proxy mainly retrieves information from the db (maybe with a few modifications for efficient retrieval)\
+
+## Max Open Connections
+
+The db can handle 100 open connections concurrently (that is the default value for postgres).\
```

Comment:
that's the max value ? for the proxy or the for the postgres? how can can i change that value?

Reply:
this is the default value set by postgres. you can increase it as you want as long as the infrastructure can handle it. 
you can configure the postgres.conf file with the desired values, or execute an update query on this value, but this requires a db restart.